renovate.json

```diff
@@ -1,3 +1,34 @@
 {
-  "$schema": "https://docs.renovatebot.com/renovate-schema.json"
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "prConcurrentLimit": 5,
+  "extends": [
+    "config:js-app",
+    "group:monorepos",
+    "group:recommended",
+    "packages:eslint",
+    "packages:unitTest",
+    "packages:tslint"
+  ],
+  "packageRules": [
+    {
+      "packagePatterns": ["spur-"],
+      "groupName": "Spur packages",
+      "description": "All packages related to spur node framework"
+    },
+    {
+      "packagePatterns": ["prettier", "-prettier"],
```

Comment:
Would `prettier` also cover `prettier-`?

Reply:
Unfortunately no. I'll have to recheck but it required it in another repo.